releases/release-5.0.3.md

```diff
@@ -0,0 +1,127 @@
+    - 修复了 `TopN` 算子下使用 `ParallelApply` 查询时卡住的问题 [#25011](https://github.com/pingcap/tidb/pull/25011)
+    - 修复了使用含有多列的前缀索引查询时出现多余结果的问题 [#24635](https://github.com/pingcap/tidb/pull/24635)
+    - 修复了操作符 `<=>` 不能正确生效的问题 [#24633](https://github.com/pingcap/tidb/pull/24633)
+    - 修复并行 `Apply` 算子的数据竞争问题 [#24345](https://github.com/pingcap/tidb/pull/24345)
+    ?- 修复在分区表上对 union IndexMerge 结果排序时出现的 `index out of range` 错误 [#24155](https://github.com/pingcap/tidb/pull/24155)
+    - 禁止设置一个未来的读时间戳 [#25761](https://github.com/pingcap/tidb/pull/25761)
+    - 修复 ODBC 类常数（例如 `{d '2020-01-01'}`）不能被用作表达式的问题 [#25577](https://github.com/pingcap/tidb/pull/25577)
+    - 修复 `SELECT DISTINCT` 被转化为 Batch Get 而导致结果不正确的问题 [#25533](https://github.com/pingcap/tidb/pull/25533)
+    - 修复无法触发将查询从 TiFlash 回退到 TiKV 的问题 [#24600](https://github.com/pingcap/tidb/pull/24600)
+    - 修复在检查 `only_full_group_by` 时的 `index-out-of-range` 错误 [#24016](https://github.com/pingcap/tidb/pull/24016)
+
++ TiKV
+
+    - 修复错误的 `tikv_raftstore_hibernated_peer_state` 监控指标 [#10431](https://github.com/tikv/tikv/pull/10431)
+    - 修复 coprocessor 中 `json_unquote()` 函数错误的参数类型 [#10424](https://github.com/tikv/tikv/pull/10424)
+    - 修复在 Backup & Restore 数据恢复期间开启 TDE 会报出文件已存在的错误 [#10421](https://github.com/tikv/tikv/pull/10421)
+    - 正常关机时跳过清理 Raftstore 的回调从而避免在某些情况下破坏事务的 ACID [#10396](https://github.com/tikv/tikv/pull/10396)
+    - 修复在 Leader 上 Replica Read 共享 Read Index 的问题 [#10391](https://github.com/tikv/tikv/pull/10391)
+    - 修复 coprocessor 转换 `DOUBLE` 到 `DOUBLE` 的错误函数 [#10388](https://github.com/tikv/tikv/pull/10388)
+
++ TiFlash
+
+    - 修复因 split 失败而不断重启的问题
+    ?- 修复无法 GC Delta 数据的潜在问题
+    - 修复在 `CAST` 函数中为非二进制字符串填充错误数据的问题
+    - 修复处理包含复杂 `GROUP BY` 列的聚合查询时结果不正确的问题
+    - 修复写入压力过大时出现进程崩溃的问题
+    - 修复右连接键不为空且左连接键可为空时进程崩溃的问题
+    - 修复 `read-index` 请求耗时长的潜在问题
+
++ Tools
+
+    + TiCDC
+
+        - 修复 TiCDC owner 在刷新 checkpoint 时异常退出的问题 [#2031](https://github.com/pingcap/ticdc/pull/2031)
+        - 修复写 MySQL 下游出错暂停时 MySQL 连接泄漏的问题 [#1946](https://github.com/pingcap/ticdc/pull/1946)
+        - 修复 TiCDC 读取 `/proc/meminfo` 失败时出现的 panic 问题 [#2024](https://github.com/pingcap/ticdc/pull/2024)
+        - 减少 TiCDC 运行时的内存使用 [#2012](https://github.com/pingcap/ticdc/pull/2012) [#1958](https://github.com/pingcap/ticdc/pull/1958)
+        - 修复 resolved ts 计算慢导致 TiCDC panic 的问题 [#2047](https://github.com/pingcap/ticdc/pull/2047)
+        - 修复 processor 潜在的死锁问题 [#2142](https://github.com/pingcap/ticdc/pull/2142)
+
+    + Backup & Restore (BR)
+
+        - 修复 BR 恢复中忽略了所有系统表的问题 [#1224](https://github.com/pingcap/br/pull/1224)
+
+    + TiDB Lightning
+
+        - 修复 TiDB Lightning 在特殊数据下 panic 的问题 [#1268](https://github.com/pingcap/br/pull/1268)
+        - 修复 TiDB Lightning 导入大文件拆分时遇到的 EOF 报错问题 [#1189](https://github.com/pingcap/br/pull/1189)
+        ?- 修复 TiDB Lightning 导入主键为 `auto_increment double` 类型表时生成极大 increment 值的问题 [#1186](https://github.com/pingcap/br/pull/1186)
```

Comment:
@lichunzhu The Chinese and English versions look inconsistent (at least literally). Could you help make the two versions look closer and clearer?

The English version: "Fix a bug that TiDB Lightning rebase wrong `auto_increment` base when the `auto_increment` field type is `FLOAT` or `DOUBLE`"

Reply:
```suggestion
        ?- 修复 TiDB Lightning 导入含 `auto_increment` 的 double/float 类型列的表时生成极大 base 值的问题 [#1186](https://github.com/pingcap/br/pull/1186)
```